lib/mapObjects/CObjectClassesHandler.h

```diff
@@ -65,6 +65,30 @@ struct DLL_LINKAGE RandomMapInfo
 	}
 };
 
+struct DLL_LINKAGE AiMapObjectID
+{
+	si32 primaryID;
+	si32 secondaryID;
+
+	AiMapObjectID(si32 primID, si32 secID) : primaryID(primID), secondaryID(secID) {};
+
+	bool operator<(const AiMapObjectID& other)
+	{
+		if(this->primaryID != other.primaryID)
+			return this->primaryID < other.primaryID;
+		else
+			return this->secondaryID < other.secondaryID;
+	}
+
+	bool operator==(const AiMapObjectID& other)
+	{
+		if(this->primaryID == other.primaryID)
+			return this->secondaryID == other.secondaryID;
+
+		return false;
```

Comment:
should this be just ` return this->primaryID == other.primaryID && this->secondaryID == other.secondaryID`?